*** Starting uWSGI 2.0.11.1 (32bit) on [Wed Sep 16 09:07:07 2015] ***
compiled with version: 5.2.0 on 09 August 2015 15:08:59
os: Linux-3.18.14-2-ARCH #1 SMP PREEMPT Thu May 28 07:19:33 MDT 2015
nodename: pi1
machine: armv7l
clock source: unix
pcre jit disabled
detected number of CPU cores: 4
current working directory: /
writing pidfile to /srv/http/app/run/.pid
detected binary path: /usr/bin/uwsgi
chdir() to /srv/http/app/htdocs
your processes number limit is 7328
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /srv/http/app/run/app.uwsgi.sock fd 3
Python version: 3.4.3 (default, Mar 26 2015, 06:22:32)  [GCC 4.9.2 20141224 (prerelease)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x23c098
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 128512 bytes (125 KB) for 1 cores
*** Operational MODE: single process ***
WSGI app 0 (mountpoint='') ready in 2 seconds on interpreter 0x23c098 pid: 24494 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 24494)
spawned uWSGI worker 1 (pid: 24497, cores: 1)
[pid: 24497|app: 0|req: 1/1] 95.204.40.111 () {36 vars in 504 bytes} [Wed Sep 16 09:08:58 2015] POST /addData => generated 209 bytes in 83 msecs (HTTP/1.1 302) 4 headers in 298 bytes (1 switches on core 0)
announcing my loyalty to the Emperor...
[pid: 24497|app: 0|req: 2/2] 78.78.15.137 () {36 vars in 503 bytes} [Wed Sep 16 09:13:22 2015] POST /addData => generated 209 bytes in 34 msecs (HTTP/1.1 302) 4 headers in 298 bytes (1 switches on core 0)
[pid: 24497|app: 0|req: 3/3] 90.236.12.38 () {36 vars in 503 bytes} [Wed Sep 16 09:17:52 2015] POST /addData => generated 209 bytes in 33 msecs (HTTP/1.1 302) 4 headers in 298 bytes (1 switches on core 0)
[pid: 24497|app: 0|req: 4/4] 90.232.157.109 () {36 vars in 505 bytes} [Wed Sep 16 09:22:36 2015] POST /addData => generated 209 bytes in 34 msecs (HTTP/1.1 302) 4 headers in 298 bytes (1 switches on core 0)
[pid: 24497|app: 0|req: 5/5] 192.168.1.13 () {36 vars in 566 bytes} [Wed Sep 16 09:26:04 2015] GET /gps => generated 2088 bytes in 252 msecs (HTTP/1.1 200) 2 headers in 81 bytes (1 switches on core 0)
[pid: 24497|app: 0|req: 6/6] 192.168.1.13 () {36 vars in 582 bytes} [Wed Sep 16 09:26:04 2015] GET /favicon.ico => generated 233 bytes in 130 msecs (HTTP/1.1 404) 2 headers in 72 bytes (1 switches on core 0)
[pid: 24497|app: 0|req: 7/7] 192.168.1.13 () {38 vars in 597 bytes} [Wed Sep 16 09:26:06 2015] GET /gps => generated 2088 bytes in 26 msecs (HTTP/1.1 200) 2 headers in 81 bytes (1 switches on core 0)
SIGINT/SIGQUIT received...killing workers...
Wed Sep 16 09:27:15 2015 - received message 0 from emperor
worker 1 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.11.1 (32bit) on [Wed Sep 16 09:27:18 2015] ***
compiled with version: 5.2.0 on 09 August 2015 15:08:59
os: Linux-3.18.14-2-ARCH #1 SMP PREEMPT Thu May 28 07:19:33 MDT 2015
nodename: pi1
machine: armv7l
clock source: unix
pcre jit disabled
detected number of CPU cores: 4
current working directory: /
writing pidfile to /srv/http/app/run/.pid
detected binary path: /usr/bin/uwsgi
chdir() to /srv/http/app/htdocs
your processes number limit is 7328
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /srv/http/app/run/app.uwsgi.sock fd 3
Python version: 3.4.3 (default, Mar 26 2015, 06:22:32)  [GCC 4.9.2 20141224 (prerelease)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0xb16098
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 128512 bytes (125 KB) for 1 cores
*** Operational MODE: single process ***
WSGI app 0 (mountpoint='') ready in 1 seconds on interpreter 0xb16098 pid: 24556 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 24556)
spawned uWSGI worker 1 (pid: 24558, cores: 1)
[pid: 24558|app: 0|req: 1/1] 192.168.1.13 () {38 vars in 597 bytes} [Wed Sep 16 09:27:20 2015] GET /gps => generated 2208 bytes in 213 msecs (HTTP/1.1 200) 2 headers in 81 bytes (1 switches on core 0)
announcing my loyalty to the Emperor...
[pid: 24558|app: 0|req: 2/2] 95.195.107.48 () {36 vars in 504 bytes} [Wed Sep 16 09:27:21 2015] POST /addData => generated 209 bytes in 67 msecs (HTTP/1.1 302) 4 headers in 298 bytes (1 switches on core 0)
SIGINT/SIGQUIT received...killing workers...
Wed Sep 16 09:27:49 2015 - received message 0 from emperor
worker 1 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.11.1 (32bit) on [Wed Sep 16 09:27:53 2015] ***
compiled with version: 5.2.0 on 09 August 2015 15:08:59
os: Linux-3.18.14-2-ARCH #1 SMP PREEMPT Thu May 28 07:19:33 MDT 2015
nodename: pi1
machine: armv7l
clock source: unix
pcre jit disabled
detected number of CPU cores: 4
current working directory: /
writing pidfile to /srv/http/app/run/.pid
detected binary path: /usr/bin/uwsgi
chdir() to /srv/http/app/htdocs
your processes number limit is 7328
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /srv/http/app/run/app.uwsgi.sock fd 3
Python version: 3.4.3 (default, Mar 26 2015, 06:22:32)  [GCC 4.9.2 20141224 (prerelease)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x58d098
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 128512 bytes (125 KB) for 1 cores
*** Operational MODE: single process ***
WSGI app 0 (mountpoint='') ready in 1 seconds on interpreter 0x58d098 pid: 24571 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 24571)
spawned uWSGI worker 1 (pid: 24573, cores: 1)
[pid: 24573|app: 0|req: 1/1] 192.168.1.13 () {38 vars in 597 bytes} [Wed Sep 16 09:27:54 2015] GET /gps => generated 2466 bytes in 192 msecs (HTTP/1.1 200) 2 headers in 81 bytes (1 switches on core 0)
announcing my loyalty to the Emperor...
2
6
621655
621657
[pid: 24573|app: 0|req: 2/2] 192.168.1.13 () {36 vars in 574 bytes} [Wed Sep 16 09:28:14 2015] GET /tempmap => generated 2794 bytes in 300 msecs (HTTP/1.1 200) 2 headers in 81 bytes (1 switches on core 0)
2
6
621655
621657
[pid: 24573|app: 0|req: 3/3] 192.168.1.13 () {38 vars in 605 bytes} [Wed Sep 16 09:28:21 2015] GET /tempmap => generated 2794 bytes in 225 msecs (HTTP/1.1 200) 2 headers in 81 bytes (1 switches on core 0)
SIGINT/SIGQUIT received...killing workers...
Wed Sep 16 09:49:25 2015 - received message 0 from emperor
worker 1 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.11.1 (32bit) on [Wed Sep 16 09:49:28 2015] ***
compiled with version: 5.2.0 on 09 August 2015 15:08:59
os: Linux-3.18.14-2-ARCH #1 SMP PREEMPT Thu May 28 07:19:33 MDT 2015
nodename: pi1
machine: armv7l
clock source: unix
pcre jit disabled
detected number of CPU cores: 4
current working directory: /
writing pidfile to /srv/http/app/run/.pid
detected binary path: /usr/bin/uwsgi
chdir() to /srv/http/app/htdocs
your processes number limit is 7328
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /srv/http/app/run/app.uwsgi.sock fd 3
Python version: 3.4.3 (default, Mar 26 2015, 06:22:32)  [GCC 4.9.2 20141224 (prerelease)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x18c4098
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 128512 bytes (125 KB) for 1 cores
*** Operational MODE: single process ***
WSGI app 0 (mountpoint='') ready in 2 seconds on interpreter 0x18c4098 pid: 24632 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 24632)
spawned uWSGI worker 1 (pid: 24634, cores: 1)
[pid: 24634|app: 0|req: 1/1] 78.79.15.43 () {36 vars in 502 bytes} [Wed Sep 16 09:51:02 2015] POST /addData => generated 209 bytes in 85 msecs (HTTP/1.1 302) 4 headers in 298 bytes (1 switches on core 0)
announcing my loyalty to the Emperor...
[pid: 24634|app: 0|req: 2/2] 90.236.201.166 () {36 vars in 505 bytes} [Wed Sep 16 09:58:32 2015] POST /addData => generated 209 bytes in 35 msecs (HTTP/1.1 302) 4 headers in 298 bytes (1 switches on core 0)
[pid: 24634|app: 0|req: 3/3] 95.203.12.228 () {36 vars in 504 bytes} [Wed Sep 16 10:00:41 2015] POST /addData => generated 209 bytes in 32 msecs (HTTP/1.1 302) 4 headers in 298 bytes (1 switches on core 0)
[pid: 24634|app: 0|req: 4/4] 95.192.190.132 () {36 vars in 505 bytes} [Wed Sep 16 10:02:59 2015] POST /addData => generated 209 bytes in 32 msecs (HTTP/1.1 302) 4 headers in 298 bytes (1 switches on core 0)
[pid: 24634|app: 0|req: 5/5] 192.168.1.13 () {38 vars in 597 bytes} [Wed Sep 16 10:04:05 2015] GET /gps => generated 3365 bytes in 250 msecs (HTTP/1.1 200) 2 headers in 81 bytes (1 switches on core 0)
[pid: 24634|app: 0|req: 6/6] 90.237.59.33 () {36 vars in 503 bytes} [Wed Sep 16 10:05:04 2015] POST /addData => generated 209 bytes in 32 msecs (HTTP/1.1 302) 4 headers in 298 bytes (1 switches on core 0)
[pid: 24634|app: 0|req: 7/7] 90.232.238.215 () {36 vars in 505 bytes} [Wed Sep 16 10:07:07 2015] POST /addData => generated 209 bytes in 32 msecs (HTTP/1.1 302) 4 headers in 298 bytes (1 switches on core 0)
[pid: 24634|app: 0|req: 8/8] 95.192.205.130 () {36 vars in 505 bytes} [Wed Sep 16 10:09:03 2015] POST /addData => generated 209 bytes in 32 msecs (HTTP/1.1 302) 4 headers in 298 bytes (1 switches on core 0)
[pid: 24634|app: 0|req: 9/9] 90.235.41.55 () {36 vars in 503 bytes} [Wed Sep 16 10:11:07 2015] POST /addData => generated 209 bytes in 32 msecs (HTTP/1.1 302) 4 headers in 298 bytes (1 switches on core 0)
[pid: 24634|app: 0|req: 10/10] 90.232.28.173 () {36 vars in 504 bytes} [Wed Sep 16 10:14:41 2015] POST /addData => generated 209 bytes in 32 msecs (HTTP/1.1 302) 4 headers in 298 bytes (1 switches on core 0)
[pid: 24634|app: 0|req: 11/11] 192.168.1.13 () {38 vars in 597 bytes} [Wed Sep 16 10:15:05 2015] GET /gps => generated 4213 bytes in 45 msecs (HTTP/1.1 200) 2 headers in 81 bytes (1 switches on core 0)
[pid: 24634|app: 0|req: 12/12] 192.168.1.13 () {38 vars in 597 bytes} [Wed Sep 16 10:16:28 2015] GET /gps => generated 1174 bytes in 17 msecs (HTTP/1.1 200) 2 headers in 81 bytes (1 switches on core 0)
[pid: 24634|app: 0|req: 13/13] 192.168.1.13 () {38 vars in 597 bytes} [Wed Sep 16 10:24:06 2015] GET /gps => generated 1174 bytes in 17 msecs (HTTP/1.1 200) 2 headers in 81 bytes (1 switches on core 0)
[pid: 24634|app: 0|req: 14/14] 192.168.1.13 () {38 vars in 597 bytes} [Wed Sep 16 10:24:07 2015] GET /gps => generated 1174 bytes in 17 msecs (HTTP/1.1 200) 2 headers in 81 bytes (1 switches on core 0)
[pid: 24634|app: 0|req: 15/15] 192.168.1.13 () {38 vars in 597 bytes} [Wed Sep 16 10:28:32 2015] GET /gps => generated 1174 bytes in 17 msecs (HTTP/1.1 200) 2 headers in 81 bytes (1 switches on core 0)
[pid: 24634|app: 0|req: 16/16] 192.168.1.13 () {38 vars in 597 bytes} [Wed Sep 16 10:29:58 2015] GET /gps => generated 1174 bytes in 17 msecs (HTTP/1.1 200) 2 headers in 81 bytes (1 switches on core 0)
[pid: 24634|app: 0|req: 17/17] 192.168.1.13 () {38 vars in 597 bytes} [Wed Sep 16 10:30:00 2015] GET /gps => generated 1174 bytes in 17 msecs (HTTP/1.1 200) 2 headers in 81 bytes (1 switches on core 0)
[pid: 24634|app: 0|req: 18/18] 192.168.1.13 () {38 vars in 597 bytes} [Wed Sep 16 10:30:01 2015] GET /gps => generated 1174 bytes in 17 msecs (HTTP/1.1 200) 2 headers in 81 bytes (1 switches on core 0)
[pid: 24634|app: 0|req: 19/19] 192.168.1.13 () {38 vars in 597 bytes} [Wed Sep 16 10:30:02 2015] GET /gps => generated 1174 bytes in 17 msecs (HTTP/1.1 200) 2 headers in 81 bytes (1 switches on core 0)
[pid: 24634|app: 0|req: 20/20] 192.168.1.13 () {38 vars in 597 bytes} [Wed Sep 16 10:30:03 2015] GET /gps => generated 1174 bytes in 17 msecs (HTTP/1.1 200) 2 headers in 81 bytes (1 switches on core 0)
[pid: 24634|app: 0|req: 21/21] 192.168.1.13 () {38 vars in 597 bytes} [Wed Sep 16 10:31:12 2015] GET /gps => generated 1174 bytes in 17 msecs (HTTP/1.1 200) 2 headers in 81 bytes (1 switches on core 0)
[pid: 24634|app: 0|req: 22/22] 78.78.59.154 () {36 vars in 503 bytes} [Wed Sep 16 10:35:47 2015] POST /addData => generated 209 bytes in 40 msecs (HTTP/1.1 302) 4 headers in 298 bytes (1 switches on core 0)
[pid: 24634|app: 0|req: 23/23] 192.168.1.13 () {38 vars in 597 bytes} [Wed Sep 16 10:35:54 2015] GET /gps => generated 1626 bytes in 22 msecs (HTTP/1.1 200) 2 headers in 81 bytes (1 switches on core 0)
SIGINT/SIGQUIT received...killing workers...
Wed Sep 16 10:41:52 2015 - received message 0 from emperor
worker 1 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.11.1 (32bit) on [Wed Sep 16 10:41:55 2015] ***
compiled with version: 5.2.0 on 09 August 2015 15:08:59
os: Linux-3.18.14-2-ARCH #1 SMP PREEMPT Thu May 28 07:19:33 MDT 2015
nodename: pi1
machine: armv7l
clock source: unix
pcre jit disabled
detected number of CPU cores: 4
current working directory: /
writing pidfile to /srv/http/app/run/.pid
detected binary path: /usr/bin/uwsgi
chdir() to /srv/http/app/htdocs
your processes number limit is 7328
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /srv/http/app/run/app.uwsgi.sock fd 3
Python version: 3.4.3 (default, Mar 26 2015, 06:22:32)  [GCC 4.9.2 20141224 (prerelease)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0xd44098
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 128512 bytes (125 KB) for 1 cores
*** Operational MODE: single process ***
WSGI app 0 (mountpoint='') ready in 2 seconds on interpreter 0xd44098 pid: 24799 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 24799)
spawned uWSGI worker 1 (pid: 24801, cores: 1)
Traceback (most recent call last):
  File "/usr/lib/python3.4/site-packages/flask/app.py", line 1836, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/lib/python3.4/site-packages/flask/app.py", line 1820, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/usr/lib/python3.4/site-packages/flask/app.py", line 1403, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3.4/site-packages/flask/_compat.py", line 33, in reraise
    raise value
  File "/usr/lib/python3.4/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/lib/python3.4/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/lib/python3.4/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3.4/site-packages/flask/_compat.py", line 33, in reraise
    raise value
  File "/usr/lib/python3.4/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/lib/python3.4/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "./flaskr.py", line 116, in show_gps_entries
    return render_template('show_gps_entries.html', positions=positions)
  File "/usr/lib/python3.4/site-packages/flask/templating.py", line 127, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/usr/lib/python3.4/site-packages/jinja2/environment.py", line 851, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/usr/lib/python3.4/site-packages/jinja2/environment.py", line 812, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/usr/lib/python3.4/site-packages/jinja2/environment.py", line 786, in _load_template
    template = self.loader.load(self, name, globals)
  File "/usr/lib/python3.4/site-packages/jinja2/loaders.py", line 125, in load
    code = environment.compile(source, name, filename)
  File "/usr/lib/python3.4/site-packages/jinja2/environment.py", line 565, in compile
    self.handle_exception(exc_info, source_hint=source_hint)
  File "/usr/lib/python3.4/site-packages/jinja2/environment.py", line 754, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3.4/site-packages/jinja2/_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "/srv/http/app/htdocs/templates/show_gps_entries.html", line 19, in template
    {{ item.value2 }},{{ item.value3 }},{{ item.value4 }},{{ item.value5 }},{{ item.value 6}},{{ 
  File "/usr/lib/python3.4/site-packages/jinja2/environment.py", line 470, in _parse
    return Parser(self, source, name, encode_filename(filename)).parse()
  File "/usr/lib/python3.4/site-packages/jinja2/parser.py", line 897, in parse
    result = nodes.Template(self.subparse(), lineno=1)
  File "/usr/lib/python3.4/site-packages/jinja2/parser.py", line 879, in subparse
    rv = self.parse_statement()
  File "/usr/lib/python3.4/site-packages/jinja2/parser.py", line 121, in parse_statement
    return getattr(self, 'parse_' + self.stream.current.value)()
  File "/usr/lib/python3.4/site-packages/jinja2/parser.py", line 231, in parse_block
    node.body = self.parse_statements(('name:endblock',), drop_needle=True)
  File "/usr/lib/python3.4/site-packages/jinja2/parser.py", line 156, in parse_statements
    result = self.subparse(end_tokens)
  File "/usr/lib/python3.4/site-packages/jinja2/parser.py", line 879, in subparse
    rv = self.parse_statement()
  File "/usr/lib/python3.4/site-packages/jinja2/parser.py", line 121, in parse_statement
    return getattr(self, 'parse_' + self.stream.current.value)()
  File "/usr/lib/python3.4/site-packages/jinja2/parser.py", line 189, in parse_for
    body = self.parse_statements(('name:endfor', 'name:else'))
  File "/usr/lib/python3.4/site-packages/jinja2/parser.py", line 156, in parse_statements
    result = self.subparse(end_tokens)
  File "/usr/lib/python3.4/site-packages/jinja2/parser.py", line 872, in subparse
    self.stream.expect('variable_end')
  File "/usr/lib/python3.4/site-packages/jinja2/lexer.py", line 374, in expect
    self.name, self.filename)
jinja2.exceptions.TemplateSyntaxError: expected token 'end of print statement', got 'integer'
[pid: 24801|app: 0|req: 1/1] 192.168.1.13 () {38 vars in 597 bytes} [Wed Sep 16 10:41:57 2015] GET /gps => generated 0 bytes in 210 msecs (HTTP/1.1 500) 0 headers in 0 bytes (0 switches on core 0)
announcing my loyalty to the Emperor...
Traceback (most recent call last):
  File "/usr/lib/python3.4/site-packages/flask/app.py", line 1836, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/lib/python3.4/site-packages/flask/app.py", line 1820, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/usr/lib/python3.4/site-packages/flask/app.py", line 1403, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3.4/site-packages/flask/_compat.py", line 33, in reraise
    raise value
  File "/usr/lib/python3.4/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/lib/python3.4/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/lib/python3.4/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3.4/site-packages/flask/_compat.py", line 33, in reraise
    raise value
  File "/usr/lib/python3.4/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/lib/python3.4/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "./flaskr.py", line 116, in show_gps_entries
    return render_template('show_gps_entries.html', positions=positions)
  File "/usr/lib/python3.4/site-packages/flask/templating.py", line 127, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/usr/lib/python3.4/site-packages/jinja2/environment.py", line 851, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/usr/lib/python3.4/site-packages/jinja2/environment.py", line 812, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/usr/lib/python3.4/site-packages/jinja2/environment.py", line 786, in _load_template
    template = self.loader.load(self, name, globals)
  File "/usr/lib/python3.4/site-packages/jinja2/loaders.py", line 125, in load
    code = environment.compile(source, name, filename)
  File "/usr/lib/python3.4/site-packages/jinja2/environment.py", line 565, in compile
    self.handle_exception(exc_info, source_hint=source_hint)
  File "/usr/lib/python3.4/site-packages/jinja2/environment.py", line 754, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3.4/site-packages/jinja2/_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "/srv/http/app/htdocs/templates/show_gps_entries.html", line 19, in template
    {{ item.value4 }},{{ item.value5 }},{{ item.value 6}},{{ item.value7 }}
  File "/usr/lib/python3.4/site-packages/jinja2/environment.py", line 470, in _parse
    return Parser(self, source, name, encode_filename(filename)).parse()
  File "/usr/lib/python3.4/site-packages/jinja2/parser.py", line 897, in parse
    result = nodes.Template(self.subparse(), lineno=1)
  File "/usr/lib/python3.4/site-packages/jinja2/parser.py", line 879, in subparse
    rv = self.parse_statement()
  File "/usr/lib/python3.4/site-packages/jinja2/parser.py", line 121, in parse_statement
    return getattr(self, 'parse_' + self.stream.current.value)()
  File "/usr/lib/python3.4/site-packages/jinja2/parser.py", line 231, in parse_block
    node.body = self.parse_statements(('name:endblock',), drop_needle=True)
  File "/usr/lib/python3.4/site-packages/jinja2/parser.py", line 156, in parse_statements
    result = self.subparse(end_tokens)
  File "/usr/lib/python3.4/site-packages/jinja2/parser.py", line 879, in subparse
    rv = self.parse_statement()
  File "/usr/lib/python3.4/site-packages/jinja2/parser.py", line 121, in parse_statement
    return getattr(self, 'parse_' + self.stream.current.value)()
  File "/usr/lib/python3.4/site-packages/jinja2/parser.py", line 189, in parse_for
    body = self.parse_statements(('name:endfor', 'name:else'))
  File "/usr/lib/python3.4/site-packages/jinja2/parser.py", line 156, in parse_statements
    result = self.subparse(end_tokens)
  File "/usr/lib/python3.4/site-packages/jinja2/parser.py", line 872, in subparse
    self.stream.expect('variable_end')
  File "/usr/lib/python3.4/site-packages/jinja2/lexer.py", line 374, in expect
    self.name, self.filename)
jinja2.exceptions.TemplateSyntaxError: expected token 'end of print statement', got 'integer'
[pid: 24801|app: 0|req: 2/2] 192.168.1.13 () {38 vars in 597 bytes} [Wed Sep 16 10:47:38 2015] GET /gps => generated 0 bytes in 208 msecs (HTTP/1.1 500) 0 headers in 0 bytes (0 switches on core 0)
SIGINT/SIGQUIT received...killing workers...
Wed Sep 16 10:49:02 2015 - received message 0 from emperor
worker 1 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.11.1 (32bit) on [Wed Sep 16 10:49:05 2015] ***
compiled with version: 5.2.0 on 09 August 2015 15:08:59
os: Linux-3.18.14-2-ARCH #1 SMP PREEMPT Thu May 28 07:19:33 MDT 2015
nodename: pi1
machine: armv7l
clock source: unix
pcre jit disabled
detected number of CPU cores: 4
current working directory: /
writing pidfile to /srv/http/app/run/.pid
detected binary path: /usr/bin/uwsgi
chdir() to /srv/http/app/htdocs
your processes number limit is 7328
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /srv/http/app/run/app.uwsgi.sock fd 3
Python version: 3.4.3 (default, Mar 26 2015, 06:22:32)  [GCC 4.9.2 20141224 (prerelease)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x4ff098
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 128512 bytes (125 KB) for 1 cores
*** Operational MODE: single process ***
  File "./flaskr.py", line 121
    return render_template('show_gps_entries.html', positions=positions)
    ^
SyntaxError: 'return' outside function
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 24833)
spawned uWSGI worker 1 (pid: 24834, cores: 1)
--- no python application found, check your startup logs for errors ---
[pid: 24834|app: -1|req: -1/1] 192.168.1.13 () {38 vars in 597 bytes} [Wed Sep 16 10:49:07 2015] GET /gps => generated 21 bytes in 3 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
announcing my loyalty to the Emperor...
--- no python application found, check your startup logs for errors ---
[pid: 24834|app: -1|req: -1/2] 192.168.1.13 () {38 vars in 597 bytes} [Wed Sep 16 10:49:09 2015] GET /gps => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 24834|app: -1|req: -1/3] 192.168.1.1 () {38 vars in 646 bytes} [Wed Sep 16 10:49:36 2015] GET /gps => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 24834|app: -1|req: -1/4] 192.168.1.1 () {38 vars in 662 bytes} [Wed Sep 16 10:49:36 2015] GET /favicon.ico => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
SIGINT/SIGQUIT received...killing workers...
Wed Sep 16 10:50:16 2015 - received message 0 from emperor
worker 1 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.11.1 (32bit) on [Wed Sep 16 10:50:19 2015] ***
compiled with version: 5.2.0 on 09 August 2015 15:08:59
os: Linux-3.18.14-2-ARCH #1 SMP PREEMPT Thu May 28 07:19:33 MDT 2015
nodename: pi1
machine: armv7l
clock source: unix
pcre jit disabled
detected number of CPU cores: 4
current working directory: /
writing pidfile to /srv/http/app/run/.pid
detected binary path: /usr/bin/uwsgi
chdir() to /srv/http/app/htdocs
your processes number limit is 7328
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /srv/http/app/run/app.uwsgi.sock fd 3
Python version: 3.4.3 (default, Mar 26 2015, 06:22:32)  [GCC 4.9.2 20141224 (prerelease)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x181d098
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 128512 bytes (125 KB) for 1 cores
*** Operational MODE: single process ***
  File "./flaskr.py", line 120
    return render_template('show_gps_entries.html', positions=positions)
    ^
SyntaxError: 'return' outside function
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 24850)
spawned uWSGI worker 1 (pid: 24851, cores: 1)
--- no python application found, check your startup logs for errors ---
[pid: 24851|app: -1|req: -1/1] 192.168.1.1 () {40 vars in 677 bytes} [Wed Sep 16 10:50:20 2015] GET /gps => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
announcing my loyalty to the Emperor...
--- no python application found, check your startup logs for errors ---
[pid: 24851|app: -1|req: -1/2] 192.168.1.1 () {40 vars in 677 bytes} [Wed Sep 16 10:50:21 2015] GET /gps => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 24851|app: -1|req: -1/3] 192.168.1.1 () {40 vars in 677 bytes} [Wed Sep 16 10:50:22 2015] GET /gps => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 24851|app: -1|req: -1/4] 192.168.1.1 () {40 vars in 677 bytes} [Wed Sep 16 10:51:20 2015] GET /gps => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 24851|app: -1|req: -1/5] 192.168.1.1 () {40 vars in 677 bytes} [Wed Sep 16 10:52:33 2015] GET /gps => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 24851|app: -1|req: -1/6] 192.168.1.1 () {40 vars in 677 bytes} [Wed Sep 16 10:52:34 2015] GET /gps => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 24851|app: -1|req: -1/7] 192.168.1.1 () {40 vars in 677 bytes} [Wed Sep 16 10:52:35 2015] GET /gps => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 24851|app: -1|req: -1/8] 192.168.1.1 () {38 vars in 654 bytes} [Wed Sep 16 10:52:43 2015] GET /heatmap => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 24851|app: -1|req: -1/9] 192.168.1.1 () {40 vars in 685 bytes} [Wed Sep 16 10:52:45 2015] GET /heatmap => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
SIGINT/SIGQUIT received...killing workers...
Wed Sep 16 10:53:14 2015 - received message 0 from emperor
worker 1 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.11.1 (32bit) on [Wed Sep 16 10:53:17 2015] ***
compiled with version: 5.2.0 on 09 August 2015 15:08:59
os: Linux-3.18.14-2-ARCH #1 SMP PREEMPT Thu May 28 07:19:33 MDT 2015
nodename: pi1
machine: armv7l
clock source: unix
pcre jit disabled
detected number of CPU cores: 4
current working directory: /
writing pidfile to /srv/http/app/run/.pid
detected binary path: /usr/bin/uwsgi
chdir() to /srv/http/app/htdocs
your processes number limit is 7328
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /srv/http/app/run/app.uwsgi.sock fd 3
Python version: 3.4.3 (default, Mar 26 2015, 06:22:32)  [GCC 4.9.2 20141224 (prerelease)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x14d9098
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 128512 bytes (125 KB) for 1 cores
*** Operational MODE: single process ***
WSGI app 0 (mountpoint='') ready in 1 seconds on interpreter 0x14d9098 pid: 24887 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 24887)
spawned uWSGI worker 1 (pid: 24889, cores: 1)
[pid: 24889|app: 0|req: 1/1] 192.168.1.1 () {40 vars in 685 bytes} [Wed Sep 16 10:53:19 2015] GET /heatmap => generated 2807 bytes in 168 msecs (HTTP/1.1 200) 2 headers in 81 bytes (1 switches on core 0)
announcing my loyalty to the Emperor...
[pid: 24889|app: 0|req: 2/2] 192.168.1.1 () {38 vars in 646 bytes} [Wed Sep 16 10:53:21 2015] GET /gps => generated 1620 bytes in 99 msecs (HTTP/1.1 200) 2 headers in 81 bytes (1 switches on core 0)
[pid: 24889|app: 0|req: 3/3] 78.78.10.87 () {36 vars in 502 bytes} [Wed Sep 16 10:53:27 2015] POST /addData => generated 209 bytes in 71 msecs (HTTP/1.1 302) 4 headers in 298 bytes (1 switches on core 0)
SIGINT/SIGQUIT received...killing workers...
Wed Sep 16 10:53:37 2015 - received message 0 from emperor
worker 1 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.11.1 (32bit) on [Wed Sep 16 10:53:40 2015] ***
compiled with version: 5.2.0 on 09 August 2015 15:08:59
os: Linux-3.18.14-2-ARCH #1 SMP PREEMPT Thu May 28 07:19:33 MDT 2015
nodename: pi1
machine: armv7l
clock source: unix
pcre jit disabled
detected number of CPU cores: 4
current working directory: /
writing pidfile to /srv/http/app/run/.pid
detected binary path: /usr/bin/uwsgi
chdir() to /srv/http/app/htdocs
your processes number limit is 7328
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /srv/http/app/run/app.uwsgi.sock fd 3
Python version: 3.4.3 (default, Mar 26 2015, 06:22:32)  [GCC 4.9.2 20141224 (prerelease)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x1975098
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 128512 bytes (125 KB) for 1 cores
*** Operational MODE: single process ***
WSGI app 0 (mountpoint='') ready in 2 seconds on interpreter 0x1975098 pid: 24902 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 24902)
spawned uWSGI worker 1 (pid: 24905, cores: 1)
[pid: 24905|app: 0|req: 1/1] 192.168.1.1 () {40 vars in 677 bytes} [Wed Sep 16 10:53:42 2015] GET /gps => generated 2124 bytes in 182 msecs (HTTP/1.1 200) 2 headers in 81 bytes (1 switches on core 0)
announcing my loyalty to the Emperor...
[pid: 24905|app: 0|req: 2/2] 192.168.1.1 () {40 vars in 677 bytes} [Wed Sep 16 10:54:58 2015] GET /gps => generated 2653 bytes in 143 msecs (HTTP/1.1 200) 2 headers in 81 bytes (1 switches on core 0)
[pid: 24905|app: 0|req: 3/3] 95.192.115.220 () {36 vars in 505 bytes} [Wed Sep 16 11:12:43 2015] POST /addData => generated 209 bytes in 73 msecs (HTTP/1.1 302) 4 headers in 298 bytes (1 switches on core 0)
[pid: 24905|app: 0|req: 4/4] 192.168.1.1 () {40 vars in 677 bytes} [Wed Sep 16 11:23:45 2015] GET /gps => generated 3420 bytes in 39 msecs (HTTP/1.1 200) 2 headers in 81 bytes (1 switches on core 0)
[pid: 24905|app: 0|req: 5/5] 192.168.1.1 () {40 vars in 677 bytes} [Wed Sep 16 11:27:57 2015] GET /gps => generated 3420 bytes in 39 msecs (HTTP/1.1 200) 2 headers in 81 bytes (1 switches on core 0)
Traceback (most recent call last):
  File "/usr/lib/python3.4/site-packages/flask/app.py", line 1836, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/lib/python3.4/site-packages/flask/app.py", line 1820, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/usr/lib/python3.4/site-packages/flask/app.py", line 1403, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3.4/site-packages/flask/_compat.py", line 33, in reraise
    raise value
  File "/usr/lib/python3.4/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/lib/python3.4/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/lib/python3.4/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3.4/site-packages/flask/_compat.py", line 33, in reraise
    raise value
  File "/usr/lib/python3.4/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/lib/python3.4/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "./flaskr.py", line 57, in show_tempmap
    value2 = list(map(int, dataTransp[2]))
ValueError: invalid literal for int() with base 10: ' 26.1'
[pid: 24905|app: 0|req: 6/6] 192.168.1.1 () {38 vars in 654 bytes} [Wed Sep 16 11:28:01 2015] GET /tempmap => generated 0 bytes in 96 msecs (HTTP/1.1 500) 0 headers in 0 bytes (0 switches on core 0)
Traceback (most recent call last):
  File "/usr/lib/python3.4/site-packages/flask/app.py", line 1836, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/lib/python3.4/site-packages/flask/app.py", line 1820, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/usr/lib/python3.4/site-packages/flask/app.py", line 1403, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3.4/site-packages/flask/_compat.py", line 33, in reraise
    raise value
  File "/usr/lib/python3.4/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/lib/python3.4/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/lib/python3.4/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3.4/site-packages/flask/_compat.py", line 33, in reraise
    raise value
  File "/usr/lib/python3.4/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/lib/python3.4/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "./flaskr.py", line 57, in show_tempmap
    value2 = list(map(int, dataTransp[2]))
ValueError: invalid literal for int() with base 10: ' 26.1'
[pid: 24905|app: 0|req: 7/7] 192.168.1.1 () {40 vars in 685 bytes} [Wed Sep 16 11:28:04 2015] GET /tempmap => generated 0 bytes in 96 msecs (HTTP/1.1 500) 0 headers in 0 bytes (0 switches on core 0)
[pid: 24905|app: 0|req: 8/8] 192.168.1.1 () {38 vars in 646 bytes} [Wed Sep 16 11:28:39 2015] GET /gps => generated 3420 bytes in 40 msecs (HTTP/1.1 200) 2 headers in 81 bytes (1 switches on core 0)
SIGINT/SIGQUIT received...killing workers...
Wed Sep 16 11:29:12 2015 - received message 0 from emperor
worker 1 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.11.1 (32bit) on [Wed Sep 16 11:29:15 2015] ***
compiled with version: 5.2.0 on 09 August 2015 15:08:59
os: Linux-3.18.14-2-ARCH #1 SMP PREEMPT Thu May 28 07:19:33 MDT 2015
nodename: pi1
machine: armv7l
clock source: unix
pcre jit disabled
detected number of CPU cores: 4
current working directory: /
writing pidfile to /srv/http/app/run/.pid
detected binary path: /usr/bin/uwsgi
chdir() to /srv/http/app/htdocs
your processes number limit is 7328
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /srv/http/app/run/app.uwsgi.sock fd 3
Python version: 3.4.3 (default, Mar 26 2015, 06:22:32)  [GCC 4.9.2 20141224 (prerelease)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x127e098
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 128512 bytes (125 KB) for 1 cores
*** Operational MODE: single process ***
WSGI app 0 (mountpoint='') ready in 2 seconds on interpreter 0x127e098 pid: 24994 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 24994)
spawned uWSGI worker 1 (pid: 24997, cores: 1)
1
4
621656
621657
[pid: 24997|app: 0|req: 1/1] 192.168.1.13 () {36 vars in 574 bytes} [Wed Sep 16 11:29:19 2015] GET /tempmap => generated 2888 bytes in 397 msecs (HTTP/1.1 200) 2 headers in 81 bytes (1 switches on core 0)
announcing my loyalty to the Emperor...
[pid: 24997|app: 0|req: 2/2] 78.78.14.219 () {36 vars in 503 bytes} [Wed Sep 16 11:32:25 2015] POST /addData => generated 209 bytes in 72 msecs (HTTP/1.1 302) 4 headers in 298 bytes (1 switches on core 0)
[pid: 24997|app: 0|req: 3/3] 78.78.48.209 () {36 vars in 503 bytes} [Wed Sep 16 12:00:42 2015] POST /addData => generated 209 bytes in 39 msecs (HTTP/1.1 302) 4 headers in 298 bytes (1 switches on core 0)
SIGINT/SIGQUIT received...killing workers...
Wed Sep 16 12:24:32 2015 - received message 0 from emperor
worker 1 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.11.1 (32bit) on [Wed Sep 16 12:24:35 2015] ***
compiled with version: 5.2.0 on 09 August 2015 15:08:59
os: Linux-3.18.14-2-ARCH #1 SMP PREEMPT Thu May 28 07:19:33 MDT 2015
nodename: pi1
machine: armv7l
clock source: unix
pcre jit disabled
detected number of CPU cores: 4
current working directory: /
writing pidfile to /srv/http/app/run/.pid
detected binary path: /usr/bin/uwsgi
chdir() to /srv/http/app/htdocs
your processes number limit is 7328
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /srv/http/app/run/app.uwsgi.sock fd 3
Python version: 3.4.3 (default, Mar 26 2015, 06:22:32)  [GCC 4.9.2 20141224 (prerelease)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x1ea8098
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 128512 bytes (125 KB) for 1 cores
*** Operational MODE: single process ***
WSGI app 0 (mountpoint='') ready in 1 seconds on interpreter 0x1ea8098 pid: 25124 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 25124)
spawned uWSGI worker 1 (pid: 25126, cores: 1)
[pid: 25126|app: 0|req: 1/1] 192.168.1.1 () {40 vars in 677 bytes} [Wed Sep 16 12:24:37 2015] GET /gps => generated 1174 bytes in 198 msecs (HTTP/1.1 200) 2 headers in 81 bytes (1 switches on core 0)
announcing my loyalty to the Emperor...
[pid: 25126|app: 0|req: 2/2] 78.78.53.183 () {36 vars in 503 bytes} [Wed Sep 16 12:30:36 2015] POST /addData => generated 209 bytes in 72 msecs (HTTP/1.1 302) 4 headers in 298 bytes (1 switches on core 0)
[pid: 25126|app: 0|req: 3/3] 192.168.1.1 () {40 vars in 677 bytes} [Wed Sep 16 12:39:33 2015] GET /gps => generated 1887 bytes in 25 msecs (HTTP/1.1 200) 2 headers in 81 bytes (1 switches on core 0)
2
5
621657
621660
[pid: 25126|app: 0|req: 4/4] 192.168.1.13 () {38 vars in 605 bytes} [Wed Sep 16 12:39:38 2015] GET /tempmap => generated 2561 bytes in 180 msecs (HTTP/1.1 200) 2 headers in 81 bytes (1 switches on core 0)
2
5
621657
621660
[pid: 25126|app: 0|req: 5/5] 192.168.1.13 () {38 vars in 605 bytes} [Wed Sep 16 12:39:49 2015] GET /tempmap => generated 2561 bytes in 104 msecs (HTTP/1.1 200) 2 headers in 81 bytes (1 switches on core 0)
[pid: 25126|app: 0|req: 6/6] 192.168.1.1 () {40 vars in 677 bytes} [Wed Sep 16 12:39:53 2015] GET /gps => generated 1887 bytes in 25 msecs (HTTP/1.1 200) 2 headers in 81 bytes (1 switches on core 0)
2
5
621657
621660
[pid: 25126|app: 0|req: 7/7] 192.168.1.13 () {38 vars in 605 bytes} [Wed Sep 16 12:40:05 2015] GET /tempmap => generated 2561 bytes in 104 msecs (HTTP/1.1 200) 2 headers in 81 bytes (1 switches on core 0)
[pid: 25126|app: 0|req: 8/8] 78.78.3.219 () {36 vars in 502 bytes} [Wed Sep 16 12:43:33 2015] POST /addData => generated 209 bytes in 40 msecs (HTTP/1.1 302) 4 headers in 298 bytes (1 switches on core 0)
4
17
621655
621660
[pid: 25126|app: 0|req: 9/9] 192.168.1.13 () {38 vars in 605 bytes} [Wed Sep 16 12:50:43 2015] GET /tempmap => generated 2653 bytes in 183 msecs (HTTP/1.1 200) 2 headers in 81 bytes (1 switches on core 0)
SIGINT/SIGQUIT received...killing workers...
Wed Sep 16 12:51:17 2015 - received message 0 from emperor
worker 1 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.11.1 (32bit) on [Wed Sep 16 12:51:20 2015] ***
compiled with version: 5.2.0 on 09 August 2015 15:08:59
os: Linux-3.18.14-2-ARCH #1 SMP PREEMPT Thu May 28 07:19:33 MDT 2015
nodename: pi1
machine: armv7l
clock source: unix
pcre jit disabled
detected number of CPU cores: 4
current working directory: /
writing pidfile to /srv/http/app/run/.pid
detected binary path: /usr/bin/uwsgi
chdir() to /srv/http/app/htdocs
your processes number limit is 7328
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /srv/http/app/run/app.uwsgi.sock fd 3
Python version: 3.4.3 (default, Mar 26 2015, 06:22:32)  [GCC 4.9.2 20141224 (prerelease)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x1d48098
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 128512 bytes (125 KB) for 1 cores
*** Operational MODE: single process ***
WSGI app 0 (mountpoint='') ready in 2 seconds on interpreter 0x1d48098 pid: 25196 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 25196)
spawned uWSGI worker 1 (pid: 25198, cores: 1)
4
17
621655
621660
[pid: 25198|app: 0|req: 1/1] 192.168.1.13 () {38 vars in 605 bytes} [Wed Sep 16 12:51:22 2015] GET /tempmap => generated 33299 bytes in 369 msecs (HTTP/1.1 200) 2 headers in 82 bytes (1 switches on core 0)
announcing my loyalty to the Emperor...
[pid: 25198|app: 0|req: 2/2] 192.168.1.1 () {40 vars in 677 bytes} [Wed Sep 16 12:51:36 2015] GET /gps => generated 2655 bytes in 139 msecs (HTTP/1.1 200) 2 headers in 81 bytes (1 switches on core 0)
[pid: 25198|app: 0|req: 3/3] 95.192.70.163 () {36 vars in 504 bytes} [Wed Sep 16 12:51:44 2015] POST /addData => generated 209 bytes in 71 msecs (HTTP/1.1 302) 4 headers in 298 bytes (1 switches on core 0)
SIGINT/SIGQUIT received...killing workers...
Wed Sep 16 12:54:45 2015 - received message 0 from emperor
worker 1 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.11.1 (32bit) on [Wed Sep 16 12:54:48 2015] ***
compiled with version: 5.2.0 on 09 August 2015 15:08:59
os: Linux-3.18.14-2-ARCH #1 SMP PREEMPT Thu May 28 07:19:33 MDT 2015
nodename: pi1
machine: armv7l
clock source: unix
pcre jit disabled
detected number of CPU cores: 4
current working directory: /
writing pidfile to /srv/http/app/run/.pid
detected binary path: /usr/bin/uwsgi
chdir() to /srv/http/app/htdocs
your processes number limit is 7328
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /srv/http/app/run/app.uwsgi.sock fd 3
Python version: 3.4.3 (default, Mar 26 2015, 06:22:32)  [GCC 4.9.2 20141224 (prerelease)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x1593098
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 128512 bytes (125 KB) for 1 cores
*** Operational MODE: single process ***
WSGI app 0 (mountpoint='') ready in 2 seconds on interpreter 0x1593098 pid: 25223 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 25223)
spawned uWSGI worker 1 (pid: 25225, cores: 1)
4
17
621655
621660
[pid: 25225|app: 0|req: 1/1] 192.168.1.13 () {38 vars in 605 bytes} [Wed Sep 16 12:54:50 2015] GET /tempmap => generated 2717 bytes in 335 msecs (HTTP/1.1 200) 2 headers in 81 bytes (1 switches on core 0)
announcing my loyalty to the Emperor...
4
17
621655
621660
[pid: 25225|app: 0|req: 2/2] 192.168.1.13 () {38 vars in 605 bytes} [Wed Sep 16 12:56:31 2015] GET /tempmap => generated 2718 bytes in 336 msecs (HTTP/1.1 200) 2 headers in 81 bytes (1 switches on core 0)
SIGINT/SIGQUIT received...killing workers...
Wed Sep 16 12:57:01 2015 - received message 0 from emperor
worker 1 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.11.1 (32bit) on [Wed Sep 16 12:57:04 2015] ***
compiled with version: 5.2.0 on 09 August 2015 15:08:59
os: Linux-3.18.14-2-ARCH #1 SMP PREEMPT Thu May 28 07:19:33 MDT 2015
nodename: pi1
machine: armv7l
clock source: unix
pcre jit disabled
detected number of CPU cores: 4
current working directory: /
writing pidfile to /srv/http/app/run/.pid
detected binary path: /usr/bin/uwsgi
chdir() to /srv/http/app/htdocs
your processes number limit is 7328
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /srv/http/app/run/app.uwsgi.sock fd 3
Python version: 3.4.3 (default, Mar 26 2015, 06:22:32)  [GCC 4.9.2 20141224 (prerelease)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x1e8c098
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 128512 bytes (125 KB) for 1 cores
*** Operational MODE: single process ***
WSGI app 0 (mountpoint='') ready in 2 seconds on interpreter 0x1e8c098 pid: 25242 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 25242)
spawned uWSGI worker 1 (pid: 25244, cores: 1)
4
17
621655
621660
[pid: 25244|app: 0|req: 1/1] 192.168.1.13 () {38 vars in 605 bytes} [Wed Sep 16 12:57:10 2015] GET /tempmap => generated 33287 bytes in 526 msecs (HTTP/1.1 200) 2 headers in 82 bytes (1 switches on core 0)
announcing my loyalty to the Emperor...
[pid: 25244|app: 0|req: 2/2] 192.168.1.1 () {38 vars in 654 bytes} [Wed Sep 16 12:57:33 2015] GET /heatmap => generated 3443 bytes in 88 msecs (HTTP/1.1 200) 2 headers in 81 bytes (1 switches on core 0)
4
17
621655
621660
[pid: 25244|app: 0|req: 3/3] 192.168.1.13 () {38 vars in 605 bytes} [Wed Sep 16 12:57:48 2015] GET /tempmap => generated 33287 bytes in 372 msecs (HTTP/1.1 200) 2 headers in 82 bytes (1 switches on core 0)
SIGINT/SIGQUIT received...killing workers...
Wed Sep 16 13:01:56 2015 - received message 0 from emperor
worker 1 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.11.1 (32bit) on [Wed Sep 16 13:01:59 2015] ***
compiled with version: 5.2.0 on 09 August 2015 15:08:59
os: Linux-3.18.14-2-ARCH #1 SMP PREEMPT Thu May 28 07:19:33 MDT 2015
nodename: pi1
machine: armv7l
clock source: unix
pcre jit disabled
detected number of CPU cores: 4
current working directory: /
writing pidfile to /srv/http/app/run/.pid
detected binary path: /usr/bin/uwsgi
chdir() to /srv/http/app/htdocs
your processes number limit is 7328
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /srv/http/app/run/app.uwsgi.sock fd 3
Python version: 3.4.3 (default, Mar 26 2015, 06:22:32)  [GCC 4.9.2 20141224 (prerelease)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x1b96098
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 128512 bytes (125 KB) for 1 cores
*** Operational MODE: single process ***
WSGI app 0 (mountpoint='') ready in 2 seconds on interpreter 0x1b96098 pid: 25268 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 25268)
spawned uWSGI worker 1 (pid: 25270, cores: 1)
Traceback (most recent call last):
  File "/usr/lib/python3.4/site-packages/flask/app.py", line 1836, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/lib/python3.4/site-packages/flask/app.py", line 1820, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/usr/lib/python3.4/site-packages/flask/app.py", line 1403, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3.4/site-packages/flask/_compat.py", line 33, in reraise
    raise value
  File "/usr/lib/python3.4/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/lib/python3.4/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/lib/python3.4/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3.4/site-packages/flask/_compat.py", line 33, in reraise
    raise value
  File "/usr/lib/python3.4/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/lib/python3.4/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "./flaskr.py", line 60, in show_tempmap
    width = long( (max(latitude)-min(latitude) )*10000)
NameError: name 'long' is not defined
[pid: 25270|app: 0|req: 1/1] 192.168.1.13 () {38 vars in 605 bytes} [Wed Sep 16 13:02:01 2015] GET /tempmap => generated 0 bytes in 74 msecs (HTTP/1.1 500) 0 headers in 0 bytes (0 switches on core 0)
announcing my loyalty to the Emperor...
SIGINT/SIGQUIT received...killing workers...
Wed Sep 16 13:04:17 2015 - received message 0 from emperor
worker 1 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.11.1 (32bit) on [Wed Sep 16 13:04:20 2015] ***
compiled with version: 5.2.0 on 09 August 2015 15:08:59
os: Linux-3.18.14-2-ARCH #1 SMP PREEMPT Thu May 28 07:19:33 MDT 2015
nodename: pi1
machine: armv7l
clock source: unix
pcre jit disabled
detected number of CPU cores: 4
current working directory: /
writing pidfile to /srv/http/app/run/.pid
detected binary path: /usr/bin/uwsgi
chdir() to /srv/http/app/htdocs
your processes number limit is 7328
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /srv/http/app/run/app.uwsgi.sock fd 3
Python version: 3.4.3 (default, Mar 26 2015, 06:22:32)  [GCC 4.9.2 20141224 (prerelease)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0xc0b098
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 128512 bytes (125 KB) for 1 cores
*** Operational MODE: single process ***
WSGI app 0 (mountpoint='') ready in 2 seconds on interpreter 0xc0b098 pid: 25289 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 25289)
spawned uWSGI worker 1 (pid: 25291, cores: 1)
4
17
621655
621660
[pid: 25291|app: 0|req: 1/1] 192.168.1.13 () {38 vars in 605 bytes} [Wed Sep 16 13:04:22 2015] GET /tempmap => generated 33441 bytes in 430 msecs (HTTP/1.1 200) 2 headers in 82 bytes (1 switches on core 0)
announcing my loyalty to the Emperor...
SIGINT/SIGQUIT received...killing workers...
Wed Sep 16 13:05:41 2015 - received message 0 from emperor
worker 1 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.11.1 (32bit) on [Wed Sep 16 13:05:45 2015] ***
compiled with version: 5.2.0 on 09 August 2015 15:08:59
os: Linux-3.18.14-2-ARCH #1 SMP PREEMPT Thu May 28 07:19:33 MDT 2015
nodename: pi1
machine: armv7l
clock source: unix
pcre jit disabled
detected number of CPU cores: 4
current working directory: /
writing pidfile to /srv/http/app/run/.pid
detected binary path: /usr/bin/uwsgi
chdir() to /srv/http/app/htdocs
your processes number limit is 7328
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /srv/http/app/run/app.uwsgi.sock fd 3
Python version: 3.4.3 (default, Mar 26 2015, 06:22:32)  [GCC 4.9.2 20141224 (prerelease)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x179e098
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 128512 bytes (125 KB) for 1 cores
*** Operational MODE: single process ***
WSGI app 0 (mountpoint='') ready in 1 seconds on interpreter 0x179e098 pid: 25306 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 25306)
spawned uWSGI worker 1 (pid: 25308, cores: 1)
4
17
621655
621660
[pid: 25308|app: 0|req: 1/1] 192.168.1.13 () {38 vars in 605 bytes} [Wed Sep 16 13:05:54 2015] GET /tempmap => generated 33299 bytes in 484 msecs (HTTP/1.1 200) 2 headers in 82 bytes (1 switches on core 0)
announcing my loyalty to the Emperor...
SIGINT/SIGQUIT received...killing workers...
Wed Sep 16 13:06:30 2015 - received message 0 from emperor
worker 1 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.11.1 (32bit) on [Wed Sep 16 13:06:33 2015] ***
compiled with version: 5.2.0 on 09 August 2015 15:08:59
os: Linux-3.18.14-2-ARCH #1 SMP PREEMPT Thu May 28 07:19:33 MDT 2015
nodename: pi1
machine: armv7l
clock source: unix
pcre jit disabled
detected number of CPU cores: 4
current working directory: /
writing pidfile to /srv/http/app/run/.pid
detected binary path: /usr/bin/uwsgi
chdir() to /srv/http/app/htdocs
your processes number limit is 7328
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /srv/http/app/run/app.uwsgi.sock fd 3
Python version: 3.4.3 (default, Mar 26 2015, 06:22:32)  [GCC 4.9.2 20141224 (prerelease)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0xca3098
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 128512 bytes (125 KB) for 1 cores
*** Operational MODE: single process ***
WSGI app 0 (mountpoint='') ready in 2 seconds on interpreter 0xca3098 pid: 25320 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 25320)
spawned uWSGI worker 1 (pid: 25322, cores: 1)
4
17
621655
621660
[pid: 25322|app: 0|req: 1/1] 192.168.1.13 () {38 vars in 605 bytes} [Wed Sep 16 13:08:22 2015] GET /tempmap => generated 33441 bytes in 502 msecs (HTTP/1.1 200) 2 headers in 82 bytes (1 switches on core 0)
announcing my loyalty to the Emperor...
[pid: 25322|app: 0|req: 2/2] 192.168.1.1 () {40 vars in 685 bytes} [Wed Sep 16 13:08:47 2015] GET /heatmap => generated 3443 bytes in 89 msecs (HTTP/1.1 200) 2 headers in 81 bytes (1 switches on core 0)
[pid: 25322|app: 0|req: 3/3] 192.168.1.1 () {40 vars in 685 bytes} [Wed Sep 16 13:08:57 2015] GET /heatmap => generated 3443 bytes in 20 msecs (HTTP/1.1 200) 2 headers in 81 bytes (1 switches on core 0)
SIGINT/SIGQUIT received...killing workers...
Wed Sep 16 13:16:00 2015 - received message 0 from emperor
worker 1 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.11.1 (32bit) on [Wed Sep 16 13:16:03 2015] ***
compiled with version: 5.2.0 on 09 August 2015 15:08:59
os: Linux-3.18.14-2-ARCH #1 SMP PREEMPT Thu May 28 07:19:33 MDT 2015
nodename: pi1
machine: armv7l
clock source: unix
pcre jit disabled
detected number of CPU cores: 4
current working directory: /
writing pidfile to /srv/http/app/run/.pid
detected binary path: /usr/bin/uwsgi
chdir() to /srv/http/app/htdocs
your processes number limit is 7328
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /srv/http/app/run/app.uwsgi.sock fd 3
Python version: 3.4.3 (default, Mar 26 2015, 06:22:32)  [GCC 4.9.2 20141224 (prerelease)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x16a7098
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 128512 bytes (125 KB) for 1 cores
*** Operational MODE: single process ***
WSGI app 0 (mountpoint='') ready in 2 seconds on interpreter 0x16a7098 pid: 25353 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 25353)
spawned uWSGI worker 1 (pid: 25355, cores: 1)
4
17
621655
621660
Traceback (most recent call last):
  File "/usr/lib/python3.4/site-packages/flask/app.py", line 1836, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/lib/python3.4/site-packages/flask/app.py", line 1820, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/usr/lib/python3.4/site-packages/flask/app.py", line 1403, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3.4/site-packages/flask/_compat.py", line 33, in reraise
    raise value
  File "/usr/lib/python3.4/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/lib/python3.4/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/lib/python3.4/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3.4/site-packages/flask/_compat.py", line 33, in reraise
    raise value
  File "/usr/lib/python3.4/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/lib/python3.4/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "./flaskr.py", line 81, in show_tempmap
    for x in range(lonCenter-150,lonCenter+50,5*latScaleFactor):
NameError: name 'latScaleFactor' is not defined
[pid: 25355|app: 0|req: 1/1] 192.168.1.13 () {38 vars in 605 bytes} [Wed Sep 16 13:16:05 2015] GET /tempmap => generated 0 bytes in 106 msecs (HTTP/1.1 500) 0 headers in 0 bytes (0 switches on core 0)
announcing my loyalty to the Emperor...
SIGINT/SIGQUIT received...killing workers...
Wed Sep 16 13:16:30 2015 - received message 0 from emperor
worker 1 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.11.1 (32bit) on [Wed Sep 16 13:16:33 2015] ***
compiled with version: 5.2.0 on 09 August 2015 15:08:59
os: Linux-3.18.14-2-ARCH #1 SMP PREEMPT Thu May 28 07:19:33 MDT 2015
nodename: pi1
machine: armv7l
clock source: unix
pcre jit disabled
detected number of CPU cores: 4
current working directory: /
writing pidfile to /srv/http/app/run/.pid
detected binary path: /usr/bin/uwsgi
chdir() to /srv/http/app/htdocs
your processes number limit is 7328
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /srv/http/app/run/app.uwsgi.sock fd 3
Python version: 3.4.3 (default, Mar 26 2015, 06:22:32)  [GCC 4.9.2 20141224 (prerelease)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x1c03098
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 128512 bytes (125 KB) for 1 cores
*** Operational MODE: single process ***
WSGI app 0 (mountpoint='') ready in 2 seconds on interpreter 0x1c03098 pid: 25368 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 25368)
spawned uWSGI worker 1 (pid: 25371, cores: 1)
4
17
621655
621660
[pid: 25371|app: 0|req: 1/1] 192.168.1.13 () {38 vars in 605 bytes} [Wed Sep 16 13:16:35 2015] GET /tempmap => generated 33383 bytes in 434 msecs (HTTP/1.1 200) 2 headers in 82 bytes (1 switches on core 0)
announcing my loyalty to the Emperor...
4
17
621655
621660
[pid: 25371|app: 0|req: 2/2] 192.168.1.13 () {38 vars in 605 bytes} [Wed Sep 16 13:16:46 2015] GET /tempmap => generated 33383 bytes in 401 msecs (HTTP/1.1 200) 2 headers in 82 bytes (1 switches on core 0)
SIGINT/SIGQUIT received...killing workers...
Wed Sep 16 13:19:12 2015 - received message 0 from emperor
worker 1 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.11.1 (32bit) on [Wed Sep 16 13:19:15 2015] ***
compiled with version: 5.2.0 on 09 August 2015 15:08:59
os: Linux-3.18.14-2-ARCH #1 SMP PREEMPT Thu May 28 07:19:33 MDT 2015
nodename: pi1
machine: armv7l
clock source: unix
pcre jit disabled
detected number of CPU cores: 4
current working directory: /
writing pidfile to /srv/http/app/run/.pid
detected binary path: /usr/bin/uwsgi
chdir() to /srv/http/app/htdocs
your processes number limit is 7328
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /srv/http/app/run/app.uwsgi.sock fd 3
Python version: 3.4.3 (default, Mar 26 2015, 06:22:32)  [GCC 4.9.2 20141224 (prerelease)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0xf9e098
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 128512 bytes (125 KB) for 1 cores
*** Operational MODE: single process ***
WSGI app 0 (mountpoint='') ready in 1 seconds on interpreter 0xf9e098 pid: 25391 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 25391)
spawned uWSGI worker 1 (pid: 25393, cores: 1)
4
17
621655
621660
[pid: 25393|app: 0|req: 1/1] 192.168.1.13 () {38 vars in 605 bytes} [Wed Sep 16 13:19:18 2015] GET /tempmap => generated 24141 bytes in 460 msecs (HTTP/1.1 200) 2 headers in 82 bytes (1 switches on core 0)
announcing my loyalty to the Emperor...
SIGINT/SIGQUIT received...killing workers...
Wed Sep 16 13:21:19 2015 - received message 0 from emperor
worker 1 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.11.1 (32bit) on [Wed Sep 16 13:21:22 2015] ***
compiled with version: 5.2.0 on 09 August 2015 15:08:59
os: Linux-3.18.14-2-ARCH #1 SMP PREEMPT Thu May 28 07:19:33 MDT 2015
nodename: pi1
machine: armv7l
clock source: unix
pcre jit disabled
detected number of CPU cores: 4
current working directory: /
writing pidfile to /srv/http/app/run/.pid
detected binary path: /usr/bin/uwsgi
chdir() to /srv/http/app/htdocs
your processes number limit is 7328
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /srv/http/app/run/app.uwsgi.sock fd 3
Python version: 3.4.3 (default, Mar 26 2015, 06:22:32)  [GCC 4.9.2 20141224 (prerelease)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x17e5098
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 128512 bytes (125 KB) for 1 cores
*** Operational MODE: single process ***
WSGI app 0 (mountpoint='') ready in 1 seconds on interpreter 0x17e5098 pid: 25408 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 25408)
spawned uWSGI worker 1 (pid: 25410, cores: 1)
4
17
621655
621660
[pid: 25410|app: 0|req: 1/1] 192.168.1.13 () {38 vars in 605 bytes} [Wed Sep 16 13:21:24 2015] GET /tempmap => generated 2631 bytes in 173 msecs (HTTP/1.1 200) 2 headers in 81 bytes (1 switches on core 0)
announcing my loyalty to the Emperor...
SIGINT/SIGQUIT received...killing workers...
Wed Sep 16 13:22:21 2015 - received message 0 from emperor
worker 1 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.11.1 (32bit) on [Wed Sep 16 13:22:24 2015] ***
compiled with version: 5.2.0 on 09 August 2015 15:08:59
os: Linux-3.18.14-2-ARCH #1 SMP PREEMPT Thu May 28 07:19:33 MDT 2015
nodename: pi1
machine: armv7l
clock source: unix
pcre jit disabled
detected number of CPU cores: 4
current working directory: /
writing pidfile to /srv/http/app/run/.pid
detected binary path: /usr/bin/uwsgi
chdir() to /srv/http/app/htdocs
your processes number limit is 7328
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /srv/http/app/run/app.uwsgi.sock fd 3
Python version: 3.4.3 (default, Mar 26 2015, 06:22:32)  [GCC 4.9.2 20141224 (prerelease)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x19a2098
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 128512 bytes (125 KB) for 1 cores
*** Operational MODE: single process ***
WSGI app 0 (mountpoint='') ready in 1 seconds on interpreter 0x19a2098 pid: 25424 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 25424)
spawned uWSGI worker 1 (pid: 25426, cores: 1)
[pid: 25426|app: 0|req: 1/1] 192.168.1.13 () {38 vars in 605 bytes} [Wed Sep 16 13:22:27 2015] GET /tempmap => generated 10868 bytes in 358 msecs (HTTP/1.1 200) 2 headers in 82 bytes (1 switches on core 0)
announcing my loyalty to the Emperor...
[pid: 25426|app: 0|req: 2/2] 192.168.1.13 () {38 vars in 605 bytes} [Wed Sep 16 13:23:23 2015] GET /tempmap => generated 10868 bytes in 125 msecs (HTTP/1.1 200) 2 headers in 82 bytes (1 switches on core 0)
[pid: 25426|app: 0|req: 3/3] 192.168.1.13 () {38 vars in 605 bytes} [Wed Sep 16 13:23:43 2015] GET /tempmap => generated 10868 bytes in 125 msecs (HTTP/1.1 200) 2 headers in 82 bytes (1 switches on core 0)
[pid: 25426|app: 0|req: 4/4] 192.168.1.13 () {38 vars in 605 bytes} [Wed Sep 16 13:24:21 2015] GET /tempmap => generated 10868 bytes in 124 msecs (HTTP/1.1 200) 2 headers in 82 bytes (1 switches on core 0)
[pid: 25426|app: 0|req: 5/5] 192.168.1.13 () {38 vars in 605 bytes} [Wed Sep 16 13:24:31 2015] GET /tempmap => generated 10868 bytes in 124 msecs (HTTP/1.1 200) 2 headers in 82 bytes (1 switches on core 0)
SIGINT/SIGQUIT received...killing workers...
Wed Sep 16 13:26:34 2015 - received message 0 from emperor
worker 1 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.11.1 (32bit) on [Wed Sep 16 13:26:38 2015] ***
compiled with version: 5.2.0 on 09 August 2015 15:08:59
os: Linux-3.18.14-2-ARCH #1 SMP PREEMPT Thu May 28 07:19:33 MDT 2015
nodename: pi1
machine: armv7l
clock source: unix
pcre jit disabled
detected number of CPU cores: 4
current working directory: /
writing pidfile to /srv/http/app/run/.pid
detected binary path: /usr/bin/uwsgi
chdir() to /srv/http/app/htdocs
your processes number limit is 7328
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /srv/http/app/run/app.uwsgi.sock fd 3
Python version: 3.4.3 (default, Mar 26 2015, 06:22:32)  [GCC 4.9.2 20141224 (prerelease)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x3ef098
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 128512 bytes (125 KB) for 1 cores
*** Operational MODE: single process ***
WSGI app 0 (mountpoint='') ready in 1 seconds on interpreter 0x3ef098 pid: 25446 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 25446)
spawned uWSGI worker 1 (pid: 25448, cores: 1)
[pid: 25448|app: 0|req: 1/1] 192.168.1.1 () {40 vars in 685 bytes} [Wed Sep 16 13:26:39 2015] GET /heatmap => generated 3443 bytes in 171 msecs (HTTP/1.1 200) 2 headers in 81 bytes (1 switches on core 0)
announcing my loyalty to the Emperor...
[pid: 25448|app: 0|req: 2/2] 192.168.1.13 () {38 vars in 605 bytes} [Wed Sep 16 13:26:41 2015] GET /tempmap => generated 11721 bytes in 211 msecs (HTTP/1.1 200) 2 headers in 82 bytes (1 switches on core 0)
